apps/frontend/src/pages/feed/libs/components/feed-activity/feed-activity.tsx

```diff
@@ -0,0 +1,48 @@
+import { Image } from "~/libs/components/components.js";
+import { DEFAULT_USER_AVATAR } from "~/libs/constants/constants.js";
```

Comment:
I deleted this avatar. Main branch did not have it.
Now we use this [one](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/blob/main/apps/frontend/src/assets/img/default-avatar.png)
```suggestion
import defaultAvatar from "~/assets/img/default-avatar.png";
```

Reply:
Thank you!